library/Icinga/Web/Navigation/Mobile/MobileConfigMenu.php

```diff
@@ -0,0 +1,226 @@
+<?php
+
+namespace Icinga\Web\Navigation\Mobile;
+
+use Icinga\Application\Icinga;
+use Icinga\Authentication\Auth;
+use Icinga\Common\HealthBadgeTrait;
+use ipl\Html\Attributes;
+use ipl\Html\BaseHtmlElement;
+use ipl\Html\HtmlElement;
+use ipl\Html\Text;
+use ipl\Web\Url;
+use ipl\Web\Widget\Icon;
+
+class MobileConfigMenu extends BaseHtmlElement
+{
+    use HealthBadgeTrait;
+
+    protected $tag = 'ul';
+
+    protected $defaultAttributes = [
+        'class' => 'nav user-nav-item',
+        'id' => 'mobile-config-menu'
+    ];
+
+    protected $children;
+
+    protected $healthBadge;
+
+    public function __construct()
+    {
+        $this->children = [
```

Comment:
I don't like that these are re-defined here. Can't you just inherit from `ConfigMenu` here? This would also avoid to re-implement `createLevel2MenuItem` for example. Just override `assemble` here.

Reply:
I love it. ✅